docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -912,9 +912,6 @@
             <para>Press the <emphasis role="bold">Next</emphasis>
             button.</para>
 
-            <para>Alternatively, you could find the IP addresses using Auto
```

Comment:
Addressed in https://github.com/hpcc-systems/HPCC-Platform/pull/9703
only not merged yet, Removed this bit to ensure the text does not get reintroduced.